website/docs/getting-started/for-engineers.md

```diff
@@ -103,30 +103,6 @@ and then use the component in your code like this:
 
 For details about how this component should be used and its API, see [the component documentation page](/components/icon).
 
-#### Using the `@hashicorp/ember-flight-icons` addon <Doc::Badge @type="warning" @size="large">Deprecated</Doc::Badge>
-
-!!! Warning
-
-This approach is now deprecated. Use the `Hds::Icon` instead.
-
-!!!
-
-The `ember-flight-icons` package is an Ember addon that provides a standalone `FlightIcon` component that can be used to render an icon as an `<svg>` HTML element.
-
-Since it's an independent package, you have to install it first:
-
-```bash
-pnpm add @hashicorp/ember-flight-icons
-```
-
-and then use the component in your code like this:
-
-```handlebars{data-execute=false}
-<FlightIcon @name="info" />
-```
-
-The [API of the component](/components/icon?tab=code#component-api) is the same as the `Hds::Icon` one.
-
 #### Deferred loading
```

Comment:
Note: depending on what we decide to do with the config object name, may need to update this.